Resources/public/css/theme/views/userprofile.css

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-userprofileview {
+    font-weight: bold;
+    color: #333;
+}
+
+.ez-view-userprofileview .ez-user-profile {
+    cursor: pointer;
+    line-height: 2em;
+    text-align: right;
+}
+
+.ez-view-userprofileview .ez-user-profile:after {
+    content: "";
```

Comment:
since we already have an icon for the arrow and that it's already used in different places, I would use it there as well for consistency.